deploy/setup.yml

```diff
@@ -0,0 +1,55 @@
+- hosts: machines
+  become: yes
+  vars:
+    main_server_public: '{{ install_dir }}/main_server/public'
+  tasks:
+
+    - name: Installing npm in main_server
+      npm:
+        path: '{{ install_dir }}/main_server'
+
+    - name: Installing npm in main_server/public/js
```

Comment:
Install node packages mentioned in package.json for user-facing website of AutolabJS